istio-1.25.advisories.yaml

```
@@ -73,3 +73,8 @@ advisories:
             componentType: go-module
             componentLocation: /usr/bin/pilot-discovery
             scanner: grype
+      - timestamp: 2025-03-14T21:56:47Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
+          note: This advisory was withdrawn due to it being a mismatched detection.
```

Small suggestion to be a bit more verbose - wasn't clear when I first read:

```suggestion
          note: |
            This advisory was withdrawn from the GHSA vulnerability DB due to being a mismatched detection.
            For more information: https://github.com/advisories/GHSA-3m87-5598-2v4f
```